src/test/resources/gradle/cucumber/multipleRequests.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Multiple Requests
+  The server should respond multiple requests
+
+  Scenario: Multiple requests
+    Given The server is running on port "33333"
+    When I send method "GET" for "/simple_get" to host at the specified port 2 times
+    Then I should receive responses with version "HTTP_1_1"
+    And Status codes 200
+    And Bodies ""
```

Comment:
Is this test valuable to us?

Reply:
No, this was leftover from the previous POST request attempt.